test/SafexMain.js

```diff
@@ -0,0 +1,255 @@
+const { ethers } = require("hardhat");
+const { use, expect } = require("chai");
+const { solidity } = require("ethereum-waffle");
+
+use(solidity);
+use(require("chai-as-promised")).should();
+
+describe("SafexMain", async () => {
+  let autoAppealableArbitrator, safexMain, acc0, acc1, acc2, acc3, acc4, acc5, acc6, acc7, acc8, acc9;
+
+  describe("SafexMain Flow", async () => {
+    it("Should deploy SafexMain", async () => {
+      [acc0, acc1, acc2, acc3, acc4, acc5, acc6, acc7, acc8, acc9] = await ethers.getSigners();
+
+      const AutoAppealableArbitrator = await ethers.getContractFactory("AutoAppealableArbitrator");
+      autoAppealableArbitrator = await AutoAppealableArbitrator.deploy(1000000000000000);
+      await autoAppealableArbitrator.deployed();
+
+      const SafexMain = await ethers.getContractFactory("SafexMain");
+      safexMain = await SafexMain.deploy(autoAppealableArbitrator.address);
+      await safexMain.deployed();
+
+      expect(await safexMain.arbitrator()).to.equal(autoAppealableArbitrator.address);
+      expect(await autoAppealableArbitrator.arbitrationCost(123)).to.equal(1000000000000000);
+    });
+
+    it("Should allow users to create a plan", async () => {
+      expect(await safexMain.plansCount()).to.equal(0);
+      expect(await safexMain.getSafexMainContractBalance()).to.equal(0);
+
+      const arbitrationFee = await autoAppealableArbitrator.arbitrationCost(123); // 1000000000000000 (0.001 eth)
+
+      // SUCCESS : create a plan
+      await safexMain
+        .connect(acc0)
+        .createPlan(
+          acc9.address,
+          "https://bafybeif52vrffdp7m2ip5f44ox552r7p477druj2w4g3r47wpuzdn7235y.ipfs.infura-ipfs.io/",
+          {
+            value: arbitrationFee.toNumber() + 1000000000000000,
```

Comment:
can't we use the ether utils to convert this `1000000000000000` to eth and later compare?